### PR TITLE
Added crab meat

### PR DESCRIPTION
**Pull request by JMWTurner**
crab meat

Never mind that crab sticks aren't actually made of crabs.

### CODE REVIEW

**Comment by Exxion** [comment]
Can you kill Norris, cut a slab of meat off him, let him revive, and repeat endlessly?
Because if so please don't fix that

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -1825,6 +1825,20 @@
 		reagents.add_reagent(CARPPHEROMONES, 3)
 		bitesize = 3
 
+/obj/item/weapon/reagent_containers/food/snacks/crab_sticks
+	name = "Not-Actually-Imitation Crab sticks"
```
Comment (PJB3005) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Should give this `\improper`

**Comment by JMWTurner** [comment]
Sadly harvesting meat off a dead critter instantly deletes it.

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (JMWTurner) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: I'll do it for that one, but all the other snacks should have it. If anyone has nothing else to do.

**Comment by Exxion** [comment]
Oh, crabs are that small?
Unfortunate

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 2]
```diff
@@ -47,6 +47,16 @@
...
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/crabmeat
+	name = "crab meat"
+	desc = "Something killed the crab, and this is the result."
+	icon_state = "raw_crab"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/crabmeat/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 3)
+	bitesize = 1
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm]: putting this in new() is weird, put it in the object definition

I know every other meat does it the way you're doing it, but as @PJB3005 once pointed out, there's no reason to maintain consistency with bad coding

**Comment by AutismSpider** [comment]
better add a way to breed crabs or some thing.

**Comment by clusterfack** [comment]
How do crabs even reproduce, scissoring?

**Comment by AutismSpider** [comment]
All species of crabs reproduce by laying eggs

**Comment by Kurfursten** [comment]
Staff of change a MoMMI to make it into a crab. :eggplant:

**Comment by FudgePucker** [comment]
Add in crab farming.
I want to make a crab farming industry as a MoMMI.

**Comment by AutismSpider** [comment]
so it be egg farming? or is it going be silly like how the space carps breed?

**Comment by Kurfursten** [comment]
It could work like roaches, which will lay their eggs only under piles of trash or food.

So make crabs only lay eggs only under loose sand items. Botany can easily acquire sand with grass anyway.

**Comment by AutismSpider** [comment]
Can we make that tiny water puddle build able form sand?

use that as crabs breeding point or spot+other suff

**Comment by Kurfursten** [comment]
Terrestrial crabs typically lay their eggs among rocks, in dirt or sand. As far as I know, only coastal and ocean crabs lay eggs in water.

Of course someone would have to sprite crab eggs before anything else.

**Comment by JMWTurner** [comment]
Let's leave crab breeding for another PR.

**Comment by PJB3005** [comment]
Don't seem to be any complaints, can't hurt anything AND I need a test case for the Discord changelog bot.

**Pull request merged**